core-js/src/main/javascript/cdf/components/BaseComponent.js

```diff
@@ -391,7 +391,7 @@ define([
               arr[val[0]] = val[1];
             });
             jXML = this.dashboard.parseXActionResult(myself, this.dashboard.urlAction(this.url, arr));
-          } else {
+          } else if(this.path!=null && this.solution!=null) {
```

Comment:
It appears that `solution` is being ignored (is (ultra-)deprecated), so it may be that some are passing a nully value, and thus it should continue working regardless of the value of solution.

The issue currently being solved, resulting in a URL with a `//` segment,  happens only when both path and action are nully.

So, I believe the condition should be `this.path != null || this.action != null`.

Reply:
I may have misunderstood the point, but I believe it should be "if(this.path && this.action)" as having one or both missing, will result in a failing request

Reply:
We are not sure what current uses exist of these functions. We're only sure that it will fail if both are nully. Hence the proposed condition. Your proposal could cause regressions on cases where only one of path or action is being specified.

Reply:
That is correct. I was confused